src/main/webapp/event-logger/index.xhtml

```diff
@@ -1,8 +1,13 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml"
-	xmlns:h="http://java.sun.com/jsf/html"
-	xmlns:f="http://java.sun.com/jsf/core"
+	xmlns:h="http://xmlns.jcp.org/jsf/html"
```

Comment:
Please incorporate this into a separate PR against the master branch. That will reduce clutter here and keep things focused on portability.

Reply:
Just notice this file, not clean all xmlns in facelets.